test/test_mutant_epitope_predictor.py

```diff
@@ -21,6 +21,9 @@
 from topiary import MutantEpitopePredictor, epitopes_to_dataframe
 from varcode import Variant, VariantCollection
 
+ensembl.download()
```

Comment:
Yeah, that's true. I do think we should solve that problem if it comes up vs. beforehand.